department-of-veterans-affairs/caseflow
# HunJerBAH/APPEALS-33 queue column fix

Resolves [APPEALS-33](https://vajira.max.gov/browse/APPEALS-33)

### Description
Updated the Queue table **Task Owner** column so that the value doesn't disappear when a filter is selected. 

### Acceptance Criteria
- [ ] Task owner column is maintained when any column is sorted

### Testing Plan
1. Sign in as an BVA user (BVADWISE)
2. Click on the Queue link
3. Click on the Queue icon in the upper left corner
![image](https://user-images.githubusercontent.com/99915461/208163381-5cc19727-32e0-49ef-9e06-b753948e2c8d.png)
4. Click switch views followed by **BVA Intake**
![image](https://user-images.githubusercontent.com/99915461/208163468-a323eefa-2a32-4416-a7ea-3f1cd076a8fd.png)
5. When the page loads, make sure all columns are displayed with proper information.
![image](https://user-images.githubusercontent.com/99915461/208163546-672f8990-ff8d-42e1-a402-12e3d5447ffb.png)
6. Click on the sort button to sort any column. After the sort, make sure the info is properly sorted and the **Task Owner** column properly displays.
![image](https://user-images.githubusercontent.com/99915461/208163687-7ab3cb75-413b-49ed-921e-162896bf2d74.png)
7. Repeat with multiple columns to make sure the table properly sorts and maintains it's information.
